tb_plugin/fe/src/apifix.d.ts

```diff
@@ -44,6 +44,10 @@ declare module './api/generated/api' {
     default_device: MemoryTableMetadata['defaultDevice']
   }
 
+  export interface MemoryEventsTableMetadata {
```

Comment:
needless to change?

Reply:
Yes, I've removed this in the latest commit.